app/components/samples/table_component.html.erb

```diff
@@ -9,7 +9,7 @@
       <thead
         class='
           text-xs uppercase text-slate-700 bg-slate-50 dark:bg-slate-700
-          dark:text-slate-400
+          dark:text-slate-300 sticky top-0 z-10 special-thead
```

Comment:
I don't think you are defining the class `special-thead` anywhere

Reply:
Removed in https://github.com/phac-nml/irida-next/pull/725/commits/9dede8b63be4018bea710a82ec1afc560feabf16